packages/react-dom/src/events/__tests__/SyntheticClipboardEvent-test.js

```diff
@@ -35,42 +47,119 @@ describe('SyntheticClipboardEvent', () => {
             items: null,
             types: null,
           };
-          var clipboardEvent = createEvent({clipboardData: clipboardData});
 
-          expect(clipboardEvent.clipboardData).toBe(clipboardData);
+          var eventHandler = event => {
+            expect(event.clipboardData).toBe(clipboardData);
+            expectedCount -= 1;
+          };
+
+          var container = document.createElement('div');
+          var div = ReactDOM.render(
+            <div
+              onCopy={eventHandler}
+              onCut={eventHandler}
+              onPaste={eventHandler}
+            />,
+            container,
+          );
+          document.body.appendChild(div);
+
+          simulateEvent(div, 'copy', clipboardData);
+          simulateEvent(div, 'cut', clipboardData);
+          simulateEvent(div, 'paste', clipboardData);
+
+          expect(expectedCount).toBe(0);
         });
       });
     });
   });
 
   describe('EventInterface', () => {
     it('normalizes properties from the Event interface', () => {
-      var target = document.createElement('div');
-      var syntheticEvent = createEvent({srcElement: target});
```

Comment:
It looks like this test specifically verifies that the synthetic event still determines the target [even if only `srcElement` was set](https://github.com/facebook/react/blob/ae7639c1165a6321c94c7c7203224c1eeeabb484/packages/react-dom/src/events/getEventTarget.js#L18). Your test doesn't verify this.